File: palindrome.java
//Take an integer input from the user and check if it is palindrome.
import java.util.*;   
class palindrome 
{  
   public static void main(String args[])  
   {  
      String original, reverse = ""; // Objects of String class  
      Scanner in = new Scanner(System.in);   
      System.out.println("Enter a number to check if it is a palindrome");  
      original = in.nextLine();   
      int length = original.length();   
      for ( int i = length - 1; i >= 0; i-- )  
         reverse = reverse + original.charAt(i);  
      if (original.equals(reverse))  
         System.out.println("Int is a palindrome.");  
      else  
         System.out.println("INt isn't a palindrome.");   
   }  
}  

File: palindrome.py
#Take an integer input from the user and check if it is palindrome.
n = int(input("enter the integer: "))
temp = n
rev = 0
while(n > 0):
    dig = n%10
    rev = rev*10 + dig
    n = n//10
if(temp == rev):
    print("its a palindrome")
else:
    print("not a palindrome")
